dhis-2/dhis-services/dhis-service-administration/src/main/java/org/hisp/dhis/maintenance/jdbc/JdbcMaintenanceStore.java

```diff
@@ -51,6 +51,11 @@
 public class JdbcMaintenanceStore
     implements MaintenanceStore
 {
+    private static final String DELETE_DELIVERY_CHANNELS = "delete from programmessage_deliverychannels where programmessagedeliverychannelsid in ";
+
+    private static final String DELETE_EMAIL = "delete from programmessage_emailaddresses where programmessageemailaddressid in ";
+
+    private static final String DELETE_PHONE_NUMBER = "delete from programmessage_phonenumbers where programmessagephonenumberid in ";
```

Comment:
Although it is good to have constants, and this class could use a complete refactor, I think it is better to prefer consistency and use inline variables instead to make it similar to the existing code.

Reply:
resolved